src/plugins/intel_npu/tests/functional/behavior/npu_driver_compiler_adapter/npu_driver_compiler_adapter_downgrade_interpolate11.cpp

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) Intel Corporation
```

Comment:
@XinWangIntel , should we remove the "npu_driver_compiler_adapter" prefix from the files if we already have a directory to include relevant tests?

Reply:
Yes, I agree